src/digisos/skjema/familie/sivilstatus/PersonSkjema.tsx

```diff
@@ -127,7 +127,6 @@ const PersonSkjema = () => {
                         <InputEnhanced
                             getName={() => FAKTUM_KEY + "_fornavn_input"}
                             id={FAKTUM_KEY + "_fornavn_input"}
-                            autoFocus={true}
```

Comment:
Hvorfor tar vi bort autofocus? :)

Reply:
jeg syns det ble litt kløn å ha den  når man navigerer med  piltaster melllom radioene fordi hvis man vil videre må man da tabbe tilbake for å kunne bla til neste radio med pil. Men skal ikke insistere på å ta den bort altså, hvis du er uenig?